src/lib/validator-regex.ts

```diff
@@ -13,7 +13,7 @@ export default class ValidatorRegex implements IValidator {
       throw Error(`property of type regex - 'parameter' should not be an array`);
     }
 
-    const regex = new RegExp(property.parameter);
+    const regex = new RegExp(property.parameter.toString());
```

Comment:
should you throw if `parameter` is an object in all validators where it doesn't make sense?

Reply:
i'll try to explain this a little better.

[here](https://github.com/medic/cht-user-management/pull/96/files#diff-b21372f5e215830ebc573ef54bac338d9c8aeefd12bbe32b944c32eaaca24b3aR14) you make an assertion if the `Select` validator is not passed parameter of type object.  

I'm saying that validators like `Name` can now get passed a parameter of type object and they won't know what to do with it...  I think all those validators should have an asserttion if the parameter is an object.

Reply:
Definitely clear 👌

Reply:
Sure, I went forward and added the assertion to validators that utilize the `parameter` property. 
For validators like `ValidatorString` and `ValidatorDob`, they don't use `parameter` at all, so I omitted the assertion in those cases.